### PR TITLE
Animals game

### PR DESCRIPTION
**Pull request by natalialqp**
Modify game according to the new changes of the game structure.
Correct images and highlighted images are lists.

### CODE REVIEW

File: ros/src/migrave_game_library/animals.py [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+        # we only take the first three elements of the list
+        possible_animals = possible_animals[0:3]
+
+        self.animal = random.choice(possible_animals)
+        if "cartoon" in self.task:
+            self.animal_image = f"{self.animal}-cartoon"
+            self.activity_parameters.correct_image = [self.animal_image]
+            self.activity_parameters.correct_image_highlighted = [f"{self.animal_image}-highlighted"]
+            # self.activity_parameters.images = [f"{x}-cartoon" for x in possible_animals]
```
Comment (alex-mitrevski) [inline ros/src/migrave_game_library/animals.py:164]: Can this line be removed?

File: ros/src/migrave_game_library/animals.py [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+        elif "black-white" in self.task:
+            self.animal_image = f"{self.animal}-black-white"
+            self.activity_parameters.correct_image = [self.animal_image]
+            self.activity_parameters.correct_image_highlighted = [f"{self.animal_image}-highlighted"]
+            # self.activity_parameters.images = [f"{x}-black-white" for x in possible_animals]
```
Comment (alex-mitrevski) [inline ros/src/migrave_game_library/animals.py:169]: Same as above.

File: ros/src/migrave_game_library/animals.py [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+        elif "draw" in self.task:
+            self.animal_image = f"{self.animal}-draw"
+            self.activity_parameters.correct_image = [self.animal_image]
+            self.activity_parameters.correct_image_highlighted = [f"{self.animal_image}-highlighted"]
+            # self.activity_parameters.images = [f"{x}-draw" for x in possible_animals]
```
Comment (alex-mitrevski) [inline ros/src/migrave_game_library/animals.py:174]: Same as above.

**Review by alex-mitrevski — CHANGES_REQUESTED** [review]

**Comment by natalialqp** [comment]
Done with the changes

**Pull request merged**